doc/api/url.md

```diff
@@ -242,6 +242,13 @@ If the value assigned to the `href` property is not a valid URL, a `TypeError`
 will be thrown.
 
 #### `url.origin`
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/XXXX
+    description: "gopher" is no longer a special scheme and `url.origin` now
+                 returns `'null'` for it.
```

Comment:
What's the trick for multiline descriptions without YAML errors?

Reply:
```suggestion
    description: The scheme "gopher" is no longer a special scheme
                 and `url.origin` now returns `'null'` for it.
```
I think it's because the parser doesn't support strings starting with quotes.